src/Docker.PowerShell/Cmdlets/GetContainerImage.cs

```diff
@@ -12,9 +13,20 @@ public class GetContainerImage : DkrCmdlet
         #region Parameters
 
         /// <summary>
+        /// The specific image name to get.
+        /// </summery>
+        [Parameter(ParameterSetName = CommonParameterSetNames.Default,
+            ValueFromPipeline = true,
+                   Position = 0)]
+        [ValidateNotNullOrEmpty]
+        [ArgumentCompleter(typeof(ImageArgumentCompleter))]
+        [Alias("ImageName")]
```

Comment:
I did this for consistency with the alias in ImageOperationCmdlet, which was done as "ImageName" to differentiate from "Name" meaning container name in New-Container.  Silly, but at least consistent.